.github/workflows/conformance.yml

```diff
@@ -61,7 +66,6 @@ jobs:
             type=edge
             type=schedule
             type=ref,event=pr
-            type=ref,event=branch,suffix=-rc,enable=${{ startsWith(github.ref, 'refs/heads/release') }}
```

Comment:
What are the impacts of removing this?

Reply:
I should have changed this instead of removing it for consistency. I'll update it.